foca/models/config.py

```diff
@@ -497,8 +528,17 @@ def set_abs_path(cls, v):  # pylint: disable=E0213
         """Resolve path relative to caller's current working directory if no
         absolute path provided.
         """
-        if not Path(v).is_absolute():
-            return str(Path.cwd() / v)
+        if(isinstance(v, str)):
```

Comment:
Okay, so for the _user's_ convenience, we allow him/her to specify a single string rather than requiring that he/she always provide a _list_ of strings, right? For us this naturally makes things more complicated. However, the user can _also_ provide a list of strings with a _single_ path only - and the behavior should be _exactly_ the same as the one expected when he/she provides a single string _outside_ of a list, right?

So what I don't understand: why don't you just check if the value is a string and _if it is_ a string, put it inside a list (I think I had started such a solution). Then after you can be sure that you are dealing with a list of strings and all the other complicated checks are unnecessary, as you will only have to deal with one case. I would _much_ prefer such a solution, as it's quite painful to see all the conditionals.